dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/interceptor/CachedTime.java

```diff
@@ -7,36 +7,28 @@
  * 
  * SPDX-License-Identifier: EPL-2.0
  *
- * Contributors:
- *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-// Liberty Change for CXF Begin
+// Liberty Change for CXF Begin - Entire class is unique to our fork, could it be contributed back? 
```

Comment:
We should open an issue for this

Reply:
Yeah in the PR I delivered to update CachedTime, it was asked for me to contribute to cxf.  I hadn't gotten to it, but would be willing to do so.

Reply:
Why did you take my changes out.  The changes in this class should be undone in a new PR.